### ISSUE TITLE
Failed to create bin.

### ISSUE THREAD

**Issue by Sllump** [kuizuo/js-deobfuscator#7]
Failed to create bin at C:\Users\opc\Documents\js-deobfuscator-main\website\node_modules\.bin\deob. ENOENT: no such file or directory, stat 'C:\Users\opc\Documents\js-deobfuscator-main\website\node_modules\deob\dist\cli.js.EXE'

Is happening at the end of pnpm i, is this an issue from my side because previous versions worked great for me.

**Comment by git87663849** [comment]
It seems like you're saying you've used an online demo for code obfuscation, and while the first file's results were impressive, the second file didn't seem to have any effect. Could you provide the code for the second file?
Sure, feel free to paste the JavaScript code here, and I'll do my best to help you understand why the obfuscation might not have worked as expected for the second file.

**Comment by kuizuo** [comment]
> The translated sentence would be: "I used your online demo for code obfuscation, and while the first file's results were impressive, the second similar file didn't seem to have any effect. I'm not sure if it's okay to paste JavaScript code here."

Of course! I have had more time recently and I am preparing to optimize this project.

**Comment by git87663849** [comment]
> > The translated sentence would be: "I used your online demo for code obfuscation, and while the first file's results were impressive, the second similar file didn't seem to have any effect. I'm not sure if it's okay to paste JavaScript code here."
> 
> Of course! I have had more time recently and I am preparing to optimize this project.

The code snippet has been sent to your email address hi@kuizuo.cn
thank you very much for your help
